src/app/shared/stache/modules/video/video.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="stache-video">
+  <iframe class="stache-video-embed" [src]="src" frameborder="0" allowfullscreen></iframe>
```

Comment:
Just thinking out loud: this component is literally just an iframe; there's nothing "video" about it. Couldn't we just call this a `<stache-iframe>`?